config/locales/en.yml

```diff
@@ -52,6 +52,13 @@ en:
         subtopic_edit:
           accordions: "Edit %{page_name} accordions"
           something: Edit something else on the
+      live_stream:
+        update:
+          success: Draft live stream url updated!
```

Comment:
Corr are we excited enough about this to give users an exclamation point?

Reply:
I didn't change the error message! That is a debate for another time.